### DISCUSSION TITLE
Partner Sharing greyed out

### DISCUSSION THREAD

**Discussion by Mat429** [immich-app/immich#592] [General]
Hi,
I see the option to partner share, but the option is greyed out.
Has this been implemented yet? Or am I doing something wrong?
Thanks

**Comment by alextran1502** [comment]
Hello,
The button is the placeholder for the feature. It is not implemented yet

**Reply by sbkg0002** [reply]
@alextran1502 is there an issue/pr to track that status of this feature?

**Reply by alextran1502** [reply]
We don't have one yet. Do you mind create one?

**Comment by finickyFalcon5** [comment]
What is partner sharing?

**Reply by alextran1502** [reply]
It is a type of sharing in which the selected partner or user has access to your selected type of photos and videos your would like to share.

For example.

- I want to share all photos and videos of person A with my wife

- I want to share all photos and videos taken at location B from time C to D with my mom

**Reply by finickyFalcon5** [reply]
Ooh. I need that feature.
I want my wife/kids to have access to all the folders on the server but I want to disable their ability to manage/delete files/folders.

**Comment by sbkg0002** [comment]
I created https://github.com/immich-app/immich/issues/931
